### PR TITLE
Fixes for print view basal rendering bugs

### PR DESCRIPTION
**Pull request by cbwebdevelopment**
See https://trello.com/c/hLh5RXez

Goes hand-in-hand with same-named branch on `blip`

### CODE REVIEW

**Comment by cbwebdevelopment** [comment]
[Daily Print View renders basal beyond the latest pump data upload time.](https://trello.com/c/hLh5RXez/5-daily-print-view-renders-basal-beyond-the-latest-pump-data-upload-time)

**Comment by cbwebdevelopment** [comment]
[PDFs do not show Data Visualization in some apps](https://trello.com/c/oVpqm14o/201-pdfs-do-not-show-data-visualization-in-some-apps)

**Comment by cbwebdevelopment** [comment]
[PDFs do not show Data Visualization in some apps](https://trello.com/c/oVpqm14o/201-pdfs-do-not-show-data-visualization-in-some-apps)

File: src/modules/print/DailyPrintView.js [hunk 1]
```diff
@@ -908,19 +908,21 @@ class DailyPrintView {
...
       }
     });
 
+    const wholeDateDeliveredPath = calculateBasalPath(basal, xScale, basalScale, {
+      endAtZero: false,
+      flushBottomOffset: -0.25,
+      isFilled: false,
+      startAtZero: false,
+    });
+    // console.log(wholeDateDeliveredPath);
```
Comment (krystophv) [inline src/modules/print/DailyPrintView.js]: Could drop this.

Comment (cbwebdevelopment) [inline src/modules/print/DailyPrintView.js]: Yeah, I'll do that.

**Pull request merged**